src/PIL/_util.py

```diff
@@ -1,41 +1,32 @@
 import os
 import sys
 
-py3 = sys.version_info.major >= 3
 py36 = sys.version_info[0:2] >= (3, 6)
 
-if py3:
 
-    def isStringType(t):
-        return isinstance(t, str)
+def isStringType(t):
```

Comment:
WDYT about removing this utility function? It is more or less just an alias of a very common Python idiom at this point and doesn't offer much abstraction.

Reply:
Good idea, will remove it. It was handy when there was a Python 2/3 difference.

Reply:
Done in 2b29d164919373337b53c6122c34fe71a2899e5c, thanks!

I can add `Co-authored-by` to these commits if you'd like?

Reply:
> I can add Co-authored-by to these commits if you'd like?

Sure, if you'd like, thanks. But there is no big deal either way. Only if convenient.

Reply:
✅